server/repository/src/migrations/v1_01_05/mod.rs

```diff
@@ -0,0 +1,37 @@
+use super::{version::Version, Migration};
+
+pub(crate) struct V1_01_05;
+
+impl Migration for V1_01_05 {
+    fn version(&self) -> Version {
+        Version::from_str("1.1.5")
+    }
+    #[cfg(feature = "postgres")]
+    fn migrate(&self, connection: &crate::StorageConnection) -> anyhow::Result<()> {
+        use crate::migrations::sql;
+        sql!(
+            connection,
+            r#"ALTER TYPE sync_api_error_code ADD VALUE IF NOT EXISTS 'API_VERSION_INCOMPATIBLE';"#
```

Comment:
there is actually a test for this that i quite like: 
https://github.com/openmsupply/open-msupply/blob/b692ba4402923e5e1b42ec5acad7714db708cdae/server/repository/src/db_diesel/sync_log_row.rs#L153-L168